tests/test_qnode_metric_tensor.py

```diff
@@ -0,0 +1,499 @@
+# Copyright 2018 Xanadu Quantum Technologies Inc.
```

Comment:
I think we should put all 'new_qnode' tests into a separate subdirectory of tests, to avoid getting confused when it comes time to deleting the old QNode/old tests.

Reply:
Done.